api/api.proto

```diff
@@ -112,8 +112,6 @@ enum IoProfile {
   IO_PROFILE_CMS = 4;
   IO_PROFILE_SYNC_SHARED = 5;
   IO_PROFILE_AUTO = 6;
-  // IO_PROFILE_BKUPSRC inherits cloudbackup's IOprofile
```

Comment:
protobuf recommends marking as obsolete instead of removing fields/constants.

Reply:
Since this is introduced recently, taking it out completely rather than marking obsolete.